### ISSUE TITLE
Fortran Variable Length Records Error in .chk

### ISSUE THREAD

**Issue by trumanmiguel** [stepan-tsirkin/wannier-berri#33]
Hi Dr. Tsirkin, 

I have attached the files as requested. For pw2wannier90.x, I used Quantum Espresso 6.6 which was compiled by the Intel 2019b toolchain as stated on: https://docs.easybuild.io/en/latest/Common-toolchains.html#intel-toolchain. 
[lsf840138.txt](https://github.com/stepan-tsirkin/wannier-berri/files/6101895/lsf840138.txt)
[In5Bi3.txt](https://github.com/stepan-tsirkin/wannier-berri/files/6101910/In5Bi3.txt)
The .chk can be found at: https://drive.google.com/file/d/1zK8bzVwovHhLKc3VW3X46TO0YIV3D8hH/view?usp=sharing

Kindly let me know if you can't download the .chk file. 

Thanks. 

Regards,
Truman

**Comment by stepan-tsirkin** [comment]
Dear @trumanmiguel  , 
I think I fixed the problem in version 0.9.0.2, could you try it and report if it works.
if not, could you also check if the following lines work on your file, and report the error. 

```
import fortio
import wannierberri
fortio.FortranFile('In5Bi3.chk', mode='r', header_dtype='int32'   ,auto_endian=True, check_file=True )
wannierberri.__w90_files.CheckPoint('In5Bi3')
```
At least on my machine it works.

**Comment by trumanmiguel** [comment]
Hi Dr. Tsirkin, 

It works, there's no problems so far. 

Thanks for fixing the issue. 

Regards, 
Truman
